### DISCUSSION TITLE
Auto-hide taskbar still shows a line of pixels at the bottom with Win10 style. Any way to fix?

### DISCUSSION THREAD

**Discussion by JDNutshell** [valinet/ExplorerPatcher#2295] [Q&A]
Hello

I'm really liking all the functions this is providing, but I'm using an OLED and I'm trying to avoid any static elements on the screen. While using the windows 10 taskbar style, there's a line of pixels along the bottom of the screen, even when viewing a blank desktop. I'm assuming this is an intended feature of the original Win10 taskbar, but I'd really like a way to hide those pixels without resorting to using the Windows 11 style taskbar (where this is not an issue).

Any help would be appreciated.

**Comment by pyrates999** [comment]
Can you show a screen shot of these pixels in the original windows 10 taskbar?

Also:

Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Reply by JDNutshell** [reply]
WIndows version is 22H2 (Build 22621.2428). Patcher version is the current new release 22621.2361.58.4

This is occurring with default settings.

Here is a link to a photo of the issue (screenshot wouldn't work well as it would be the very bottom line of the screenshot)
https://imgur.com/a/I40Uc6G

As you can see, that line of grey at the bottom is what I'm talking about. The raised section of that grey is indicating the position of the currently focused window on the taskbar.

I vaguely remember this being a default feature of hiding the taskbar on Win10, so I don't even consider it a bug or anything, I'm just looking for a way to modify things to prevent it from showing all the time.

**Reply by pyrates999** [reply]
Ok, that is windows 11 then.  Thank you.

Do you have any other applications installed that modify the UI of windows?

**Reply by JDNutshell** [reply]
The only other programs I use that affect windows functionality/UI are DIsplayFusion and Fences (from Stardock). Fences doesn't modify the taskbar, and it's occurring when DisplayFusion is uninstalled.

I have also verified that it's not an issue of pixel shifting on the oled screen, as this is happening with my standard LED monitors as well.

**Reply by pyrates999** [reply]
Can you uninstall EP, reboot, and see if it is still happening?

**Reply by JDNutshell** [reply]
It goes away the moment EP is uninstalled, stays gone after rebooting, and immediately shows up again once EP is reinstalled. Switching to win11 style with EP gets rid of it, switching back to Win10 style brings it back.

I have no reason to believe this is anything other than intended behaviour, and I was just looking for a simple work around, so I'm going to move on from this.

**Comment by N173M43R** [comment]
I can confirm that this is still a present bug. Other programs do fully hide this strip, so something must be funky within explorer patcher itself.
